auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -378,41 +283,12 @@
                     <v-btn
                       color="primary"
                       class="open-action-btn"
-                      :class="{ 'active': item.nrNumber }"
                       :data-test="getIndexedTag('view-continuation-button', item.id)"
                       @click="view(item.id)"
                     >
                       {{ getButtonLabel(item.status) }}
                     </v-btn>
                   </span>
-                  <!-- More Actions Menu -->
```

Comment:
Where is the requirement to remove this?

Reply:
That might have been a design oversight. Ask Jacqueline to be sure.

(Initially, opening the NR was added so staff could extend them for FE filings. FE continuation authorizations no longer exist but maybe we should keep it anyway?)

Reply:
Sure, confirming with Jacqueline

Reply:
Jacqueline said: Let's keep it for now since it's currently implemented that way. I'll ask the staff tomorrow and get back to you.  
Let's wait until tmr :)

Reply:
I'm not clear whether "that way" means the original implementation or the new design. Are you OK waiting until tomorrow afternoon for an answer before finishing this ticket?

Reply:
Yes sure, I will wait until tmr afternoon

Reply:
Conclusion: keep the more action menu and button to link to NameX
Reverted it back:
![image](https://github.com/user-attachments/assets/a6f2bcbb-3920-4346-97b8-fa06b28bc6c2)

Reply:
Resolving this conversation.